orca-core/src/main/groovy/com/netflix/spinnaker/orca/batch/adapters/RetryableTaskTasklet.groovy

```diff
@@ -61,9 +64,23 @@ class RetryableTaskTasklet extends TaskTasklet {
 
   @Override
   protected TaskResult doExecuteTask(Stage stage, ChunkContext chunkContext) {
-    def now = clock.millis()
-    def startTime = chunkContext.stepContext.getStepExecution().startTime.time
+    if (stage.execution.status == ExecutionStatus.PAUSED) {
+      // If the pipeline is PAUSED, ensure that the current stage and all running tasks are PAUSED
+      stage.self.status = ExecutionStatus.PAUSED
+      stage.self.tasks.findAll { it.status == ExecutionStatus.RUNNING }.each {
+        it.status = ExecutionStatus.PAUSED
+      }
+      return new DefaultTaskResult(ExecutionStatus.PAUSED)
```

Comment:
Personally, I'd say `no` it shouldn't auto-pause downstream pipelines. 

Seems like an edge case (right now) and i'm not sure the complexity is warranted. I don't think there's anything that would stop us from implementing it tho.

Happy to entertain feedback?